localstack-core/localstack/services/events/models.py

```diff
@@ -38,7 +38,12 @@
     CrossRegionAttribute,
     LocalAttribute,
 )
-from localstack.utils.aws.arns import events_archive_arn, events_replay_arn
+from localstack.utils.aws.arns import (
+    event_bus_arn,
```

Comment:
nit: intentionally `event_` vs `events_` prefix

Reply:
Yes, internally everything is called event_ .... but externally anything related to event bridge uses the shorthand events_ ... here particular only related to defining the respective ARNs for event_bridge